ovs/lib/helpers/storagedriver/container.py

```diff
@@ -0,0 +1,2 @@
+class Container(object):
```

Comment:
Whats the use of this container here?

Reply:
The only thing it brings is inheriting from object (which is done implicitly anyway)